guides/plugins/plugins/administration/adding-snippets.md

```diff
@@ -69,6 +70,23 @@ The same `$tc` helper function can be used in the templates to access translatio
 
 {% endraw %}
 
+Another feature of `$tc` is pluralization. Using a `|` in snippets, you can provide translations dependending on a number. The first part would be shown as the singular expression, while the second takes care of your plural cases.
```

Comment:
```suggestion
Another feature of `$tc` is pluralization. Use a `|` in the snippet to provide translations depending on the number. The first part shows singular expressions, while the second takes care of plural cases.
```

Reply:
I guess "singular expression" is right, since he means the grammatical singular and there's just one singular expression right there